build.gradle.kts

```diff
@@ -189,13 +191,35 @@ tasks {
         password = environment("PRIVATE_KEY_PASSWORD")
     }
 
+    withType<PrepareSandboxTask> {
+        val vendor = project.findProperty("buf.protobuf.vendor")?.toString()
+            ?: System.getenv("BUF_PROTOBUF_VENDOR") ?: "official"
+        logger.info("Current Protobuf Plugin vendor: $vendor")
+        val disabledPluginsFile = project.layout.buildDirectory.dir(configDir).get().file("disabled_plugins.txt").asFile
```

Comment:
Sorry for the previous suggestion - you'll need to modify the `file` variable below to use the new name as well.